lib/web_ui/lib/src/engine/canvaskit/embedded_views.dart

```diff
@@ -109,30 +109,39 @@ class HtmlViewEmbedder {
     _viewsToRecomposite.add(viewId);
   }
 
+  /// Record that a picture recorder is needed for [picture] to be measured.
+  void prerollPicture(PictureLayer picture) {
+    final CkPictureRecorder pictureRecorder = CkPictureRecorder();
+    pictureRecorder.beginRecording(ui.Offset.zero & _frameSize.toSize());
+    _context.pictureRecordersCreatedDuringPreroll[picture] = pictureRecorder;
+  }
+
+  /// Returns the picture recorder canvas that was created to measure [picture].
+  CkCanvas getPictureRecorderFor(PictureLayer picture) {
```

Comment:
Should this be called `getCanvasFor` since it returns a canvas rather than a picture recorder?

Reply:
done